content/en/community/hackathons/2023-05-porto/_index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Porto 2023
+date: 2023-04-12T09:30:00+02:00
+draft: false
+collapsible: true
+description: Porto Unikraft Hackathon, May 10-11, 2023
+---
+
+## Porto Unikraft Hackathon
+
+Unikraft, [Faculdade de Engenharia Universidade do Porto](https://sigarra.up.pt/feup/en/web_page.Inicial) and University of Porto Faculty of Engineering ACM Student Chapter come together to organize the **Unikraft Porto Hackathon** to be held on Wednesday and Thursday, May 10-11, 2023.
+
+The hackathon will take place as an in-person event at [Faculdade de Engenharia Universidade do Porto](https://sigarra.up.pt/feup/en/web_page.Inicial).
+The full address is: [Rua Dr. Roberto Frias, s/n, 4200-465 Porto, Portugal](https://goo.gl/maps/k9zQPvJWuGskFfBc9).
+
+Support information and discussions will take place on [Discord](http://bit.ly/UnikraftDiscord) on the `#hack-porto23` channel.
+
+### Registration
+
+To take part in the hackathon please fill this [registration form](https://forms.gle/CiVMJmBXHm38WgFY9) by **Thursday, May 4, 2023, 11pm WEST**.
+
+Please bring your own laptop.
+It's best if you have a native Linux installed on your laptop.
+Otherwise, please install [this virtual machine](https://drive.google.com/file/d/1u5DtN5kMPWxBU8UdBfnZ7DNRP2n6oiTy/view?usp=share_link).
+
+### People
+
+The hosts of the hackathon are:
+
+* [João Cardoso](https://sigarra.up.pt/feup/en/func_geral.formview?p_codigo=449856)
+* [Pedro Diniz](https://sigarra.up.pt/feup/pt/func_geral.formview?p_codigo=672712)
+* [Pedro Souto](https://sigarra.up.pt/feup/en/func_geral.formview?p_codigo=238172)
+* [João Mendes Moreira](https://dei.fe.up.pt/mdse/portfolio/director/)
+* [Tiago Filipe Castro Viana](neacm@fe.up.pt)
+
+As part of the Unikraft community, [Ștefan Jumărea](https://github.com/StefanJum) and [Edi Vintilă](https://github.com/eduardvintila) will be present on-site, with other community members providing support online, on Discord.
+
+### Schedule
+
+#### Wednesday, May 10, 2023
+
+| Time (WEST)   | Session                                                              |
+| ------------- | -------------------------------------------------------------------- |
+| 10:00 - 10:15 | Overview of Unikernels and Unikraft                                  |
+| 10:15 - 11:15 | Introduction to Unikraft: Building and Running Standard Applications |
+| 11:15 - 11:30 | Break                                                                |
+| 11:30 - 12:45 | Unikraft Internals: Building, Configuring, Using Libraries           |
+| 12:45 - 14:00 | Lunch                                                                |
+| 14:00 - 15:00 | Running (Complex) Applications in Unikraft                           |
+| 15:00 - 16:00 | Porting Applications to Unikraft                                     |
+| 16:00 - 16:15 | Break                                                                |
+| 16:15 - 17:45 | Porting Applications to Unikraft                                     |
```

Comment:
There are two of  `Porting Applications to Unikraft `. Should one of them be `Binary Compatibility`?

Reply:
They're both going to be on that. But the term `Porting Applications to Unikraft` is more encompassing.

Reply:
Or anything related to application porting, actually. That's why I would leave it as such, to fit most approaches.